src/yb/yql/cql/ql/util/cql_message.h

```diff
@@ -96,6 +96,7 @@ class CQLMessage {
   static constexpr Flags kTracingFlag       = 0x02;
   static constexpr Flags kCustomPayloadFlag = 0x04; // Since V4
   static constexpr Flags kWarningFlag       = 0x08; // Since V4
+  static constexpr Flags kMetadataFlag      = 0x80;
```

Comment:
Please add a comment something like : "// Not specified by CQL protocol - YB custom flag"